Sources/Build/BuildManifest/LLBuildManifestBuilder.swift

```diff
@@ -322,26 +324,26 @@ extension ResolvedTarget {
     }
 
     public func getLLBuildTargetName(config: String) -> String {
-        "\(name)-\(config).module"
+        "\(self.name)-\(config)\(self.buildTriple.suffix).module"
```

Comment:
I wonder if whether we should have a generic build product suffix here since there can be other degrees of separation for build products in the future. 

This can happen in a follow up PR, though.